react/features/base/conference/middleware.js

```diff
@@ -33,6 +33,9 @@ MiddlewareRegistry.register(store => next => action => {
     case CONNECTION_ESTABLISHED:
         return _connectionEstablished(store, next, action);
 
+    case CONFERENCE_JOINED:
```

Comment:
can't we do this on WILL_JOIN? If there are no other participants, the user may remain in that state until another participant joins.